op-service/log/cli.go

```diff
@@ -270,6 +277,7 @@ func ReadCLIConfig(ctx *cli.Context) CLIConfig {
 // ReadTestCLIConfig reads the CLI config from flags and environment variables into a CLIConfig.
 // flag.Parse() must be called before calling this function.
 func ReadTestCLIConfig() CLIConfig {
+	*flFormat = "logfmtms" // override the default cli format of text
```

Comment:
I didn't want to change the `DefaultCLIConfig`, as other folks might be using the current default value, but at the same time I think it makes sense to have tests use `logfmtms`.